*** Util.java.orig	Fri Jun 29 10:08:05 2007
--- Util.java	Fri Jun 29 10:09:44 2007
***************
*** 24,30 ****
  import java.net.InetAddress;
  import java.net.UnknownHostException;
  
- import org.globus.common.CoGProperties;
  
  
  public class Util {
--- 24,29 ----
***************
*** 387,412 ****
          return str.toString();
      }
      
-     /**
-      * Returns the ip address of the local machine. 
-      * If the 'ip' system property is defined it is returned,
-      * otherwise, the local ip address is lookup using the
-      * <code>InetAddress</code> class. In case the lookup
-      * fails, the address 127.0.0.1 is returned.
-      *
-      * @return local ip address
-      */
-     public static String getLocalHostAddress() {
-         String ipAddr = CoGProperties.getDefault().getIPAddress();
-         if (ipAddr == null) {
-             try {
-                 return InetAddress.getLocalHost().getHostAddress();
-             } catch (UnknownHostException e) {
-                 return "127.0.0.1";
-             }
-         } else {
-             return ipAddr;
-         }
-     }
-     
  }
--- 386,389 ----
